.eslintrc.js

```diff
@@ -3,23 +3,19 @@ module.exports = {
   env: {
     node: true
   },
-  extends: ["plugin:vue/essential", "eslint:recommended", "@vue/prettier"],
+  extends: ['plugin:vue/essential', 'standard'],
   parserOptions: {
-    parser: "babel-eslint"
-  },
-  rules: {
-    "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
-    "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off"
```

Comment:
Is there a reason to remove these?

Reply:
They're not really doing anything. We're not stripping consoles in production mode in the code, so this just add some warnings during the build that no one is reading.

Maybe `no-debugger` could stay.